### PR TITLE
bugfix namespace in bendgridcanvas

### PR DESCRIPTION
**Pull request by lyrra**
- [ x] I signed the [CLA](https://musescore.org/en/cla)

### CODE REVIEW

**Comment by lyrra** [comment]
Not sure why the checks can compile fine without this PR:
```
 [1210/1326] Building CXX object src/inspector/CMakeFiles/inspector.dir/Unity/unity_1_cxx.cxx.o
```
Whereas on my site it fails (debian bookworm with qt6):
```
builds/Linux-Qt-usr-Make-Debug/src/inspector/CMakeFiles/inspector.dir/Unity/unity_1_cxx.cxx:25:
src/inspector/view/widgets/bendgridcanvas.cpp: In member function ‘void mu::inspector::BendGridCanvas::setPointList(QVariant)’:
src/inspector/view/widgets/bendgridcanvas.cpp:337:9: error: reference to ‘AccessibleItem’ is ambiguous
```
The two headers making it ambiguous are:
```
src/framework/ui/view/qmlaccessible.h:68:7: note: candidates are: ‘class mu::ui::AccessibleItem’
   68 | class AccessibleItem : public QObject, public QQmlParserStatus, public accessibility::IAccessible
      |       ^~~~~~~~~~~~~~

src/engraving/accessibility/accessibleitem.h:36:7: note:                 ‘class mu::engraving::AccessibleItem’
   36 | class AccessibleItem : public accessibility::IAccessible, public std::enable_shared_from_this<AccessibleItem>
      |       ^~~~~~~~~~~~~~ 
```

File: src/inspector/view/widgets/bendgridcanvas.cpp [hunk 1]
```diff
@@ -965,7 +965,7 @@ void BendGridCanvas::setAccessibleParent(ui::AccessibleItem* parent)
 
     m_accessibleParent = parent;
 
-    for (AccessibleItem* item : m_pointsAccessibleItems) {
+    for (ui::AccessibleItem* item : m_pointsAccessibleItems) {
```
Comment (Jojo-Schmitz) [inline src/inspector/view/widgets/bendgridcanvas.cpp:968]: no `mu::` here?

Comment (Jojo-Schmitz) [reply src/inspector/view/widgets/bendgridcanvas.cpp:968]: I guess the others anin't needed either

Comment (lyrra) [reply src/inspector/view/widgets/bendgridcanvas.cpp:968]: Thanks! updated..

**Review by cbjeukendrup — APPROVED** [review]

**Pull request merged**